content/learn/book/ecs/exclusive-world-access/_index.md

```diff
@@ -1,14 +1,124 @@
+
+Like with queries, you can call [`get::<C>`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get) and [`get_mut::<C>`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get_mut) to access component data of a particular sort on a given entity. 
+If you want access to *all* of the data on an entity, use [`get_entity`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get_entity) and [`get_entity_mut](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get_entity_mut) (or their faster but riskier siblings [entity](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.entity) and [`entity_mut`]), along with various [`EntityRef`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.EntityRef.html) methods.
+
+You can create new queries using [`query`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.query) and [`query_filtered`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.query_filtered), using the former when you only have one type parameter and the latter when you want to use the second filtering type parameter of standard queries as well.
+
+### Accessing multiple parts of the `World` simultaneously
+
+When working with non-trivial exclusive `World` logic, you're likely to run into cases where you need mutable access to more than one part of the `World` at once.
+This tends to make the compiler quite unhappy, but you can use [`World::cell`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.bundles) and `World::resource_scope`(https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.resource_scope) to allow for carefully shared mutable access.
+
+TODO: explain how `cell` and `resource_scope` works.
+TODO: add `WorldCell` example
+
+### Accessing system parameters with `World` access
+
+Occasionally, you may find yourself reaching for convenient system parameters (like `EventReader` and `EventWriter`) while you have exclusive world access.
+We can call these directly, using the same syntax as we use in systems, using the [`SystemState`](https://docs.rs/bevy/latest/bevy/ecs/system/struct.SystemState) type.
+
+```rust
+use bevy::prelude::*;
+use bevy::ecs::SystemState;
+use bevy::app::AppExit;
+
+let mut world = World::new();
+
+// Extract all of the relevant parameters from the world, as if it were a system
+let mut system_state: SystemState<(Query<&Transform, With<Camera>>, Commands, EventWriter<AppExit>>)> =
+            SystemState::new(&mut world);
+
+// Use .get() if you only need read-access for less restrictive ownership constraints
+let (camera_transform_query, mut commands, mut app_exit_event_writer) = system_state.get_mut(&world);
+
+// You can then work with the extracted system parameters however you wish
+app_exit_event_writer.send(AppExit);
+```
+
+### Manually running systems
```

Comment:
If https://github.com/bevyengine/bevy/pull/2427 lands in time for 0.6, we should use that here instead.

It also yeets .system for the methods in question!